framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3470,6 +3470,16 @@ public final AnnotatedExecutableType getAnnotatedType(ExecutableElement elt) {
         return (AnnotatedExecutableType) getAnnotatedType((Element) elt);
     }
 
+    /**
+     * See {@link #getAnnotatedType(Tree)}.
```

Comment:
Can you expand on when to use this version instead of plain `getAnnotatedType`?
Should external clients ever use this method instead of the plain method? Or is this purely for framework internal use? Could we change the method to `protected`?
I see why this method is in ATF instead of in GATF, but it is a bit odd to talk about "refinement" on the ATF level, where we don't have refinement.
Is there a different name that would make sense on this level? Or push it down to GATF and do an `instanceof` check on the use site?

Reply:
Yes, I agree with pushing it down to GATF and do an `instanceof` there. ATF doesn't do anything flow-sensitive.